src/types/fixed.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  isType (identifier) {
+    let mXn = identifier.substr(5)
+
+    // Default to 256*80
+    if (!mXn || mXn.indexOf('x') === -1) mXn = '256x80'
```

Comment:
What's the point of the second part of the condition here? I don't think that things like `fixedanythingelse` should default to `fixed256x80`, and I guess this is what would happen with that condition, right?

Reply:
that's correct, is there a specific behavior you expect ? I'm now implementing it to return false just conditions like you mentioned

Reply:
Yes, I think false should be the correct return value for those irregular cases. Thanks!